.github/workflows/deploy-backend-dev.yml

```diff
@@ -4,9 +4,6 @@ on:
   push:
     branches:
       - dev
-  pull_request:
-    branches:
```

Comment:
question: what is the reason for deleting this code?

Reply:
Deployment in dev was launched during a PR even in draft so I only put the push on the branch.